test/integration/parachain/staging/sequential/redeem.test.ts

```diff
@@ -99,7 +99,18 @@ describe("redeem", () => {
     }).timeout(3 * 60000);
 
     it("should issue and request redeem", async () => {
-        for (const [vault_1_id, vault_2_id] of collateralTickerToVaultIdsMap.values()) {
+        // "usual" scope with hard coded collateral currency (or currencies)
+        const vaultsInScope = Array.from(collateralTickerToVaultIdsMap.values());
+
+        // add a pair of aUSD vaults to the list if aUSD has been registered
+        const aUSD = await getAUSDForeignAsset(interBtcAPI.assetRegistry);
+        if (aUSD !== undefined) {
+            const vault_1_id_ausd = newVaultId(api, vault_1.address, aUSD, wrappedCurrency);
+            const vault_2_id_ausd = newVaultId(api, vault_2.address, aUSD, wrappedCurrency);
+            vaultsInScope.push([vault_1_id_ausd, vault_2_id_ausd]);
```

Comment:
As I understand it, you're not running an actual aUSD vault binary, and instead just registering one. If so, why does `issueAndRedeem` succeed?

Reply:
This is correct vault client behaviour so resolving